src/com/epam/javacourse/homework2/HomeWork.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.javacourse.homework2;
+
+import java.util.Arrays;
+
+public class HomeWork {
+    public static void main(String[] args) {
+        //Fibonacci
+        Fibonacci fibonacci = new Fibonacci();
+        fibonacci.getFibonacciRow(10);
+
+        //Palindrome
+        Palindrome palindrome = new Palindrome();
+        System.out.println("\nIs Palindrome: " + palindrome.isPalindrome(121));
```

Comment:
Is this `\n` required?

Reply:
No, but it makes the messages in the console look better:
Fibonacci Series:
0 1 1 2 3 5 8 13 21 34 
Is Palindrome: true

In comparison with
Fibonacci Series:
0 1 1 2 3 5 8 13 21 34 Is Palindrome: true